src/kOS/Function/Persistence.cs

```diff
@@ -6,9 +6,136 @@
 using kOS.Safe.Utilities;
 using kOS.Serialization;
 using System;
+using KSP.IO;
+using kOS.Safe;
+using kOS.Safe.Compilation;
+using System.Collections.Generic;
 
 namespace kOS.Function
 {
+    /*
+     * A couple of syntaxes from kRISC.tpg were deprecated when subdirectories where introduced. It will be possible to
+     * remove these function below as well any metions of delete/rename file/rename volume/copy from kRISC.tpg in the future.
+     */
+    [Function("copy_deprecated")]
+    public class FunctionCopyDeprecated : FunctionBase
+    {
+        public override void Execute(SharedObjects shared)
+        {
+            PopValueAssert(shared, true);
+            PopValueAssert(shared, true);
+            PopValueAssert(shared, true);
+            AssertArgBottomAndConsume(shared);
+
+            throw new KOSDeprecationException("1.0.0", "`COPY FILENAME FROM VOLUMEID.` syntax", "`COPYPATH(FROMPATH, TOPATH)`", string.Empty);
```

Comment:
This isn't what "deprecation" is (maybe we should rename this exception to ``KOSObsoletionException``.)
"Deprecation" means it still works but it's frowned upon.  It's a warning that it's *going* to become obsoleted later on, but hasn't *yet*.

What an actual "deprecation" would do is throw a warning message on the screen but then redirect the call into the new technique so it still doesn't break the old scripts yet, but just annoys people using them enough to nag them to make the change.